### PR TITLE
feat: Add redirect to project user feedback page for Sentry 10

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/views/userFeedback/projectUserFeedback.jsx [hunk 1]
```diff
@@ -50,6 +52,13 @@ const ProjectUserFeedback = createReactClass({
   },
 
   componentWillMount() {
+    // Redirect any Sentry 10 user that has followed an old link and ended up here
+    const {organization, location, params: {orgId}} = this.props;
+    const hasSentry10 = new Set(organization.features).has('sentry10');
+    if (hasSentry10) {
+      browserHistory.replace(`/organizations/${orgId}/user-feedback/${location.search}`);
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/userFeedback/projectUserFeedback.jsx]: Do we want to set the project?

Comment (markstory) [reply src/sentry/static/sentry/app/views/userFeedback/projectUserFeedback.jsx]: Keeping the projectid in the filter would be better.

**Comment by benvinegar** [comment]
Was there a reason we didn't do like a catch-all on the Server? (vs doing it all client side)

**Comment by lynnagara** [comment]
I don't think it makes much difference, this will potentially catch any broken in app links which won't hit the server

**Review by markstory — APPROVED** [review]

**Pull request merged**